html/index.html

```diff
@@ -1,50 +1,77 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>welcome to the first task</title>
-</head>
-<body>
-    <header>
-        EKCEL FARMS LIMITED
-    </header>
-    <nav>
-        <a href="#">HOME</a>
-        <a href="#">ABOUT</a>
-        <a href="#">CONTACT US</a>
-    </nav>
-    <section>
-        <h1>MISSION STATEMENT</h1>
-        <p>To Deliver Value To All Stakeholders Including Shareholders, Employees and Our Host Communities Through High Yield On Investment, Employee Growth, Corporate Responsibility, Operational Excellence, Cutting-Edge Technology and Continual Civic Engagement.</p>
-        <h1>VISSION STATEMENT</h1>
-        <p>Our Vision Is To Become The Number One Player In The Agro-Allied Processing Industry
-            (Tomato And Cassava) In Nigeria And West Africa.</p>
-            <img src="../image/tomato.jfif" alt="tomato photo">
-            <h1>Ekcel Farms Limited (Ekcel Farms) is a wholly owned subsidiary of Oilserv Limited. The thought to diversify into the Agricultural sector of the economy was the brainchild of an astute entrepreneur and industrialist – Engr. (Sir) Emeka Okwuosa. The project is aimed at facilitating a massive agricultural revolution in Nigeria and the sub-region which in return foster an enormous economic transformation through employment generation, elimination of agricultural losses due to the current shortfall in processing facilities and encourage independent farming through an organised out-growers scheme.
```

Comment:
only one h1 tag is required within the section tag and it has been implemented on line 21